pandas/tests/arrays/categorical/test_dtypes.py

```diff
@@ -176,6 +178,13 @@ def test_astype_category(self, dtype_ordered, cat_ordered):
             expected = cat
             tm.assert_categorical_equal(result, expected)
 
+    def test_astype_object_datetime_categories(self):
+        # GH#40754
+        cat = Categorical(to_datetime(["2021-03-27", NaT], format="%Y-%m-%d"))
+        result = cat.astype(object)
+        expected = np.array(["2021-03-27", np.nan], dtype="object")
```

Comment:
shouldnt the first entry be a Timestamp?

Reply:
Yep, thx, got bit confused here